src/datamodel/quest/test/test_finished_quest.rs

```diff
@@ -0,0 +1,37 @@
+use crate::datamodel::character::{Merlin, Mordred, Percival};
+use crate::datamodel::quest::{FinishedQuest, Quest, QuestObjective};
+use crate::datamodel::Player;
+
+#[test]
+fn test_finished_quest_creation() {
+    let (jimi, pato, volan) = get_players();
+    let _finished_quest: FinishedQuest = FinishedQuest {
+        won: true,
+        quest: Quest {
+            objective: QuestObjective {
+                num_players: 2,
+                required_successes: 2,
+            },
+            proposer: &pato,
+            members: vec![&jimi, &volan],
+        },
+    };
+
+    assert!(true);
+}
+
+fn get_players() -> (Player, Player, Player) {
```

Comment:
Could you please move this function up to the beginning of the file?
I know, it looks like a superfluous remark but I think that grouping tests' tools  at the beginning of a module is a good practice for further refactors.

Reply:
I am actually used to the opposite convention: most important things are usually first, so always public methods go first, and all private methods, helper functions and implementation details go at the end. So you can open a module/test file and see what the interface and public things are, and then scroll to the bottom to see implementation details. I'm happy to change it, but I do think we should keep consistency across the whole project.

Reply:
Here's an example from the Rust book where [mock functions for tests](https://doc.rust-lang.org/book/ch15-05-interior-mutability.html#a-use-case-for-interior-mutability-mock-objects) are presented. 
So, I support the adoption of this convention.

Reply:
Sounds good then. However, I have moved the function to a new `test_utils` file, so it is the same.